[BD-21] Link to official feature toggle annotation docs in README (#67)
edx/edx-toggles: scripts/README.rst

these components with information regarding their purpose, default values and
planned expiration dates.

> The annotation format of feature toggles is documented `here <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.

Reviewer: Please Include more of the sentence in the link text for accessibility.

Author: :ok_hand:

> The annotation format of feature toggles is `documented in this How To <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.

Reviewer: Better, but the link still doesn’t say what the how to is.

```suggestion
See the `How To with annotation format of feature toggles <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__.
```
